va/va.go

```diff
@@ -575,51 +584,50 @@ func (va *ValidationAuthorityImpl) validateTLSSNI01WithZName(ctx context.Context
 	return validationRecords, probs.Unauthorized(errText)
 }
 
-func (va *ValidationAuthorityImpl) getTLSSNICerts(
+func (va *ValidationAuthorityImpl) getTLSCerts(
 	ctx context.Context,
 	hostPort string,
 	identifier core.AcmeIdentifier,
 	challenge core.Challenge,
-	zName string,
-) ([]*x509.Certificate, *probs.ProblemDetails) {
-	va.log.Info(fmt.Sprintf("%s [%s] Attempting to validate for %s %s", challenge.Type, identifier, hostPort, zName))
-	conn, err := tlsDial(ctx, hostPort, zName)
+	config *tls.Config,
+) ([]*x509.Certificate, *tls.ConnectionState, *probs.ProblemDetails) {
+	va.log.Info(fmt.Sprintf("%s [%s] Attempting to validate for %s %s", challenge.Type, identifier, hostPort, config.ServerName))
+	config.InsecureSkipVerify = true
```

Comment:
There wasn't a comment on this field of the config when it existed in `tlsDial` but I think it would be nice to add one now that clarifies why its being used in this context. Seeing this is a red flag in most other instances.